adapters/http-base/src/main/java/org/eclipse/hono/adapter/http/impl/VertxBasedHttpProtocolAdapter.java

```diff
@@ -49,6 +50,10 @@
  */
 public final class VertxBasedHttpProtocolAdapter extends AbstractVertxBasedHttpProtocolAdapter<HttpProtocolAdapterProperties> {
```

Comment:
It would be good to add an integration test here in `HttpTestBase`, similar to the other `testUploadMessageFailsFor*` methods there.